index.js

```diff
@@ -16,43 +52,57 @@ module.exports = {
       name: options.name,
 
       defaultConfig: {
-        configuredServices: {
-          bugsnag: {
-            url: 'http://notify.bugsnag.com/deploy'
+        configuredServices: function(context) {
+          return {
+            bugsnag: {
+              url: 'http://notify.bugsnag.com/deploy',
+              method: 'POST',
+              headers: {},
+              body: function() {
+                var apiKey = this.apiKey;
+
+                if (!apiKey) { return; }
```

Comment:
This would make it difficult to provide preconfigured services that notify on one hook per default (I.e. bugsnag). With this falsy value approach we just return a falsy value and the service won't be notified. If we errored out all users would have to opt-out of bugsnag which seems kind of annoying to me